uber/site_sections/common.py

```diff
@@ -0,0 +1,7 @@
+from uber.common import *
+
+
+@all_renderable()
+class Root:
+    def index(self):
+        return render('index.html')
```

Comment:
It would be more idiomatic to just move the ``index.html`` template into the ``uber/templates/common`` directory and then we could just say ``return {}`` and not need to specify the filename.